wp-content/themes/core/components/blocks/content_loop/Content_Loop_Controller.php

```diff
@@ -130,8 +130,27 @@ public function get_posts_card_args( string $layout = Card_Controller::STYLE_PLA
 				],
 			];
 
-			// CASE: If not Inline Card Style
-			if ( $layout !== Card_Controller::STYLE_INLINE ) {
+			$card_cta =
+				[
+					Link_Controller::CONTENT => __( 'Read More', 'tribe' ),
+					Link_Controller::URL     => $link['url'],
+					Link_Controller::TARGET  => $link['target'],
+					Link_Controller::CLASSES => [
+						'u-hidden',
+					],
+					Link_Controller::ATTRS   => [
+						// These attrs provide the most screen reader accessible link.
+						'id'               => $uuid . '-link',
+						'aria-labelledby'  => $uuid . '-title',
+						'aria-describedby' => $uuid . '-link',
+						// Sets this link as the card's click-within target link.
+						'data-js'          => 'target-link',
+					],
+				];
+
+			// CASE: If not Inline Card Style and is the featured layout
+			
+			if ( $layout !== Card_Controller::STYLE_INLINE || $this->layout !== 'layout_feature') {
```

Comment:
I'd prefer to see a separate card style added for the "big" featured card rather than an exception/one-off here. There's an excellent chance that we'll reuse that large card style in other contexts.